docs/concepts/access-control-concept.md

```diff
@@ -6,16 +6,16 @@ sidebar_position: 1
 
 ## Overview
 
-Lit Protocol can be used with your storage provider of choice for secure data encryption. With Lit, you can encrypt and decrypt content client-side, defining specific rules for access using [access control conditions (ACCs)](../sdk/access-control/evm/basic-examples).
+Lit Protocol can be used with your storage provider of choice for secure data encryption. With Lit, you can encrypt and decrypt content client-side, defining specific rules for access using [access control conditions (ACCs)](../sdk/access-control/evm/basic-examples.md).
```

Comment:
hey @Aaryan-R-S are you sure these are broken? I am on the docs right now and don't have any issues with any of these links? 

i am not sure that adding the 'md' is necessary

Reply:
yeah i agree, the .md shouldn't have any effect. it should work with or without the .md. so i don't think these changes are needed.

Reply:
Hi @a1ttech and @glitch003! 
You are right. They don't have any effect and I'm undoing them. 
Just keeping only those changes which are required.